web/src/features/trace/components/SpanDetailsList/SpanDetails/SpanDetails.tsx

```diff
@@ -83,6 +83,13 @@ export const SpanDetails = ({ span, expanded, onChange }: SpanDetailsProps) => {
     }
   }, [expanded]);
 
+  let errorMessage = "";
```

Comment:
I would take this piece of code to a function something like
getErrorMessage and use it in the jsx